.env-sample

```diff
@@ -0,0 +1,2 @@
+login=
```

Comment:
please add the new credentials in the already existing config file settings.ini and add constant in settings.py which will make the credentials available in the app

Reply:
Done!